src/compressed_tensors/quantization/lifecycle/apply.py

```diff
@@ -136,10 +143,24 @@ def apply_quantization_config(model: Module, config: QuantizationConfig) -> Dict
             continue  # layer matches ignore list, continue
         targets = find_name_or_class_matches(name, submodule, target_to_scheme)
         if targets:
+            scheme = _scheme_from_targets(target_to_scheme, targets, name)
+            if run_compressed:
+                weight_shape = submodule.weight.shape
+                device = next(submodule.parameters()).device
+                format = config.format
+                compressed_linear = CompressedLinear(
```

Comment:
while we only target Linears right now, we should probably check that we're overriding a Linear regardless here